spec/pubid_iso/identifier_spec.rb

```diff
@@ -647,6 +647,33 @@ module Pubid::Iso
       it_behaves_like "converts pubid to pubid"
     end
 
+    context "ISO/R 93-3:1969" do
+      let(:pubid) { "ISO/R 93-3:1969" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/R 93/1-1963" do
+      let(:original) { "ISO/R 93/1-1963" }
+      let(:pubid) { "ISO/R 93-1-1963" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/R 300/III-1968" do
+      let(:original) { "ISO/R 300/III-1968" }
+      let(:pubid) { "ISO/R 300-III-1968" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/R 657/IV" do
+      let(:original) { "ISO/R 657/IV" }
+      let(:pubid) { "ISO/R 657-IV" }
```

Comment:
The formal (new style) pubid is actually "[ISO/R 657-4:1969](https://www.iso.org/standard/4795.html)"